src/main/java/org/trigon/hardware/misc/leds/AddressableLEDStrip.java

```diff
@@ -65,46 +65,47 @@ void clearLEDColors() {
     }
 
     @Override
-    void blink(Color firstColor, Color secondColor, double blinkingIntervalSeconds) {
+    void blink(Color firstColor, double speed) {
+        double correctedSpeed = 1 - speed;
         double currentTime = Timer.getFPGATimestamp();
-        if (currentTime - lastLEDAnimationChangeTime > blinkingIntervalSeconds) {
+        if (currentTime - lastLEDAnimationChangeTime > correctedSpeed) {
             lastLEDAnimationChangeTime = currentTime;
             isLEDAnimationChanged = !isLEDAnimationChanged;
         }
-        if (isLEDAnimationChanged)
+        if (isLEDAnimationChanged) {
             staticColor(firstColor);
-        else
-            staticColor(secondColor);
+            return;
+        }
+        clearLEDColors();
     }
 
     @Override
     void staticColor(Color color) {
-        currentAnimation = () -> staticColor(color);
         setLEDColors(color, 0, numberOfLEDs - 1);
     }
 
     @Override
-    void breathe(Color color, int breathingLEDs, double cycleTimeSeconds, boolean inverted, LarsonAnimation.BounceMode bounceMode) {
-        inverted = this.inverted != inverted;
+    void breathe(Color color, int breathingLEDs, double speed, boolean inverted, LarsonAnimation.BounceMode bounceMode) {
```

Comment:
Why all the `speed` and `speed - 1` stuff?

Reply:
Because in CANdle, the higher the speed, the faster the LEDs move, and the way I implemented it in AddressableLED is to have it be the LED MoveLEDSeconds, but that would mean that the higher the number the lower the speed, So I just reversed it by doing `1 - speed`. For example, if the speed is 0.8, after doing `1 - speed` it'll be 0.2. So it'll still be the higher the number the faster it is.